src/pages/index.mdx

```diff
@@ -43,13 +43,29 @@ import Link from '@docusaurus/Link'
     </HeroActions>
   </HeroInfo>
   <HeroAsset>
-    <video autoPlay loop muted autoPlay playsInline>
+    <video autoPlay loop muted playsInline>
       <source src="/hero/hero-video.mov" type='video/mp4; codecs="hvc1"' />
       <source src="/hero/hero-video.webm" type="video/webm" />
     </video>
   </HeroAsset>
 </Hero>
 
+<Box top={{ xs: 144, lg: 190 }}>
+  <CallToActionSection
+    title="Scalable, Reliable, and Secure Decentralized Communication"
+    columns={1}
+    border={false}
+    align="center"
+    description={<>
+      Waku delivers a unified technology stack addressing the unique challenges of decentralized peer-to-peer communication, offering turnkey reliability, scalability, and censorship resistance for your applications. Built on libp2p, Waku enhances its capabilities to ensure optimal performance in diverse network conditions.<br /><br />
```

Comment:
This is not what I meant when I said I wanted an updated based on the explanation series.

What I want, is a writer to ingest the explanation series content, and revisit the home page, so that what is highlight in explanation series is present from the moment you open the page.

Which means reworkign the "Features" section and potentially the title and intro of the page.